docs/Developing/Queues-and-defer.md

```diff
@@ -73,6 +73,26 @@ def do_something():
 defer(do_something, _target="tasks-io", _url="/_ah/queue/deferred_do_something", _queue="do-something-queue")
 ```
 
+### ⚠️ Pitfalls and Warnings with `defer` ⚠️
+
+`defer` allows for executing code across service boundaries. For instance - a task (a `Callable` function in Python) can be deferred from `web` but executed on a different service by specifying a different service with the `_target` parameter. Cross-service code execution can be dangerous due to missing imports or code on the executing service. Ex: The `Callable` function might depend on an import from the service it originated on (in our instance, `web`) that is not available on the executing service. When using `defer` for enqueueing work on a different service, be mindful the code will not be executing in the same environment it was enqueued from.
+
+The safer approach for deferring cross-service work is to create a HTTP endpoint on the target service and use [`enqueue`](#Enqueued Tasks) to call the HTTP endpoint at a later date. The HTTP request will be made on the specified service (or by default the enqueueing service), but the work will always be executed on the target service.
+
+## Enqueued Tasks (`enqueue`)
+
+`enqueue` is a drop-in replacement for [google.appengine.api.taskqueue.taskqueue.add](https://cloud.google.com/appengine/docs/standard/python/refdocs/google.appengine.api.taskqueue.taskqueue#google.appengine.api.taskqueue.taskqueue.add). `enqueue` enqueues a HTTP request to be run later while automatically managing dispatching to the proper client based on the environment, getting a queue for a specified queue name, etc. Using `enqueue` is recommended for cross-service requests. `enqueue` manages executing a task with the proper parameters both in development and in production.
```

Comment:
- [ ] This needs some more information. We should be able to expand on some of this stuff. Maybe talk about the headers?